src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -243,13 +243,27 @@ Func<object>	DefenseStep()
 	
 	var RatTargetNext = SetRatTarget?.OrderBy(target => target?.DistanceMax ?? int.MaxValue)?.FirstOrDefault();
 	
+	//Populating array to check if drones are fighting or not
+	Sanderling.Interface.MemoryStruct.IUIElementText[] droneArray = Sanderling.MemoryMeasurementParsed?.Value?.WindowDroneView?.FirstOrDefault()?.LabelText.ToArray();
```

Comment:
@Viir Sanderling says WindowDroneView does not contain a definition for ListView.
I wrote it like this:
`Sanderling.MemoryMeasurementParsed?.Value?.WindowDroneView?.ListView?;`